app/server/tests/routes/index.test.ts

```diff
@@ -30,7 +30,7 @@ describe("registerRoutes", () => {
     } as any;
 
     it("registers all expected routes", () => {
-        const router = registerRoutes(mockApp);
+        const router = registerRoutes(mockApp) as unknown as typeof mockRouter;
```

Comment:
forcing type because we know it will return the `mockRouter` in this test so just helping typescript along